floris/tools/optimization/__init__.py

```diff
@@ -1 +1 @@
-from . import other, scipy, pyoptsparse, yaw_optimization
+from . import other, scipy, pyoptsparse, yaw_optimization, layout_optimization
```

Comment:
Would we want to remove the `scipy` and `pyoptsparse` folder with this pull request, assuming `yaw_optimization` and `layout_optimization` fully replace those functionalities?

Reply:
> @bayc does this pull request work with existing examples or require new ones?

Just added and updated example @paulf81 !

Reply:
Not all of the functionality has been ported over, so I have moved the folders into a legacy folder until we can finish bringing everything over.